### DISCUSSION TITLE
I can't access my account.

### DISCUSSION THREAD

**Discussion by ghost** [community/community#46235] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Hello, this is my friend's github account. My account "xingkongqwq" was stolen by a stupid bastard. He changed my email password. He also posted a lot of content about game plug-ins in my account. Please delete the content in the repository that does not belong to me, restore my account, and punish him.

**Comment by MostafaWaleed0** [comment]
The first step is to report the issue to GitHub Support by opening a support ticket. In the ticket, explain the situation and request that the account be restored and any unauthorized content be removed.

**Reply by ghost** [reply]
How do I contact GitHub support? Please be more detailed. This is my first contact with support. Thank you

**Reply by radar-blip** [reply]
Here you go: [Support Page](https://support.github.com/request)
Probably under `Account or billing issues`

**Reply by ghost** [reply]
> Here you go: [Support Page](https://support.github.com/request) Probably under `Account or billing issues`

Thank you. I'll try it.
